### PR TITLE
Added dockerize and deployment stages to pipeline

### PR DESCRIPTION
**Pull request by alexanderjervis**
added, templated, and parameterized the dockerize and deploy stages in the pipeline. Also moved discord webhook job underneath deployment job so that it is the final job of the pipeline. Docker and AKS are currently using personal resources for testing until we get project resources to replace them.

### CODE REVIEW

**Comment by escalonn** [comment]
~@MDarmawan ^ again~ @alexanderjervis , avoid commit messages that trigger false github integration behavior

File: devops/azure-pipelines/azure-pipelines.yml [hunk 1]
```diff
@@ -18,11 +18,27 @@ stages:
...
+          containerRegistry: 'project-3-docker-hub'
+          repository: 'dajpearl/forge-backend'
+          imageTag: '${{ variables.Build.SourceVersion }}'
+          dockerPath: '**/gradle.Dockerfile'
+  - stage: Deploy
+    jobs:
+      - template: ./kubernetes-template.yml
+        parameters:
+          jobName: 'Deploy'
+          serviceConnection: 'Alex-Dev-Cluster'
```
Comment (escalonn) [inline devops/azure-pipelines/azure-pipelines.yml]: https://github.com/1058-Sophia-Gavrila-Forge/Portfolio-Frontend/pull/10#discussion_r674195973

File: devops/kube/monolith-manifest.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    app: forge-frontend-backend
+  ports:
+  - port: 3000
+    name: frontend
+    protocol: TCP
+  - port: 8081
+    name: backend
+    protocol: TCP
+
+
```
Comment (escalonn) [inline devops/kube/monolith-manifest.yml:54]: random blank lines in this file; i wouldn't reject it over just this, but it does show sloppiness

**Review by escalonn — CHANGES_REQUESTED** [review]

**Review by escalonn — APPROVED** [review]

**Review by MDarmawan — APPROVED** [review]
Things that could be fixed next iteration: using ACR, maybe using a [deploy job](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/deployment-jobs) to support a more advanced deployment strategy (i.e. canary)
Also, since we no longer need the monolith-manifest file, we should delete that...

**Comment by escalonn** [comment]
> Things that could be fixed next iteration: using ACR, maybe using a [deploy job](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/deployment-jobs) to support a more advanced deployment strategy (i.e. canary)
> Also, since we no longer need the monolith-manifest file, we should delete that...

if these things aren't work items, they should be. if you're not 100% that one would be the right choice, then phrase the work item as "evaluate ___"

**Pull request merged**